mne/epochs.py

```diff
@@ -1728,7 +1749,17 @@ def _get_data(
             del drop_log
 
             self._bad_dropped = True
-            logger.info("%d bad epochs dropped" % (n_events - len(good_idx)))
+            n_bads_dropped = n_events - len(good_idx)
+            logger.info(f"{n_bads_dropped} bad epochs dropped")
+
+            if n_bads_dropped == n_events:
+                warn(
+                    "All epochs were dropped!\n"
+                    "You might need to alter reject/flat-criteria "
+                    "or drop bad channels to avoid this.\n"
+                    "Tip: With Epochs.plot_drop_log() you can see which "
```

Comment:
```suggestion
                    "or drop bad channels to avoid this. "
                    "You can use Epochs.plot_drop_log() to see which "
```